.travis/install.sh

```diff
@@ -7,7 +7,7 @@ if [[ "$(uname -s)" == 'Darwin' ]]; then
     sw_vers
     brew update || brew update
 
-    brew outdated openssl || brew upgrade openssl
+    brew install openssl@1.1
```

Comment:
Yeah I'll re-add it once I see it all work (which it looks like it's going to). PS linkers are the worst.